tests/parser/UDQTests.cpp

```diff
@@ -2769,34 +2769,34 @@ TSTEP
     {
         const auto& udq = schedule.getUDQConfig(0);
         const auto& def = udq.define("FU_TIME");
-        BOOST_CHECK( udq_state.define(def.keyword(), def.status()));
+        BOOST_CHECK( udq_state.define(def.status()));
         udq_state.add_define(0, def.keyword(), result);
     }
     {
         const auto& udq = schedule.getUDQConfig(1);
         const auto& def = udq.define("FU_TIME");
-        BOOST_CHECK( !udq_state.define(def.keyword(), def.status()));
+        BOOST_CHECK( !udq_state.define(def.status()));
     }
     {
         const auto& udq = schedule.getUDQConfig(2);
         const auto& def = udq.define("FU_TIME");
-        BOOST_CHECK( !udq_state.define(def.keyword(), def.status()));
+        BOOST_CHECK( !udq_state.define(def.status()));
     }
     {
         const auto& udq = schedule.getUDQConfig(3);
         const auto& def = udq.define("FU_TIME");
-        BOOST_CHECK( udq_state.define(def.keyword(), def.status()));
+        BOOST_CHECK( udq_state.define(def.status()));
         udq_state.add_define(3, def.keyword(), result);
     }
     {
         const auto& udq = schedule.getUDQConfig(4);
         const auto& def = udq.define("FU_TIME");
-        BOOST_CHECK( !udq_state.define(def.keyword(), def.status()));
+        BOOST_CHECK( udq_state.define(def.status()));
```

Comment:
Why is it reasonable that the state changes here (`define() -> false` :arrow_right: `define() -> true`)?

Reply:
Okay, so that means we're technically shifting from testing the expected/desired behaviour to testing what our implementation actually does.  Do you agree that in a real simulation run, a setup of the form
```
UDQ
   UPDATE FU_TIME NEXT /
/

TSTEP -- N
1 /

TSTEP -- N + 1
1 /
```
should have `define() == true` at step `N`, but `define() == false` at step `N + 1`?

Reply:
I agree, and I think I see your point - the test doesn't really make much sense as it stands now. Maybe just delete this one, as it is not something for the parser alone..?

Reply:
> Maybe just delete this one, as it is not something for the parser alone..?

Agreed. The dynamic state object is very much affected by multiple parties, so we can't know its behaviour statically. At most, the report steps provide upper bounds for when the NEXT step will happen.

Reply:
Great, thanks. Updated now.